eval/topdown.go

```diff
@@ -979,6 +964,96 @@ func plugValue(v opalog.Value, bindings *hashMap) opalog.Value {
 	}
 }
 
+func topDownQueryCompleteDoc(store Storage, rules []*opalog.Rule) (interface{}, error) {
+
+	if len(rules) > 1 {
+		return nil, fmt.Errorf("multiple conflicting rules: %v", rules[0].Name)
+	}
```

Comment:
Seems we'll (eventually) want to support multiple rules even for complete docs.  For example we might want to compute a value depending on certain top-level switches.
prod = false
p = 3 :- prod
p = 4 :- not prod